conf/routes

```diff
@@ -107,6 +107,7 @@ GET         /editions-api/issues/:id/summary                     controllers.Edi
 GET         /editions-api/issues/:id/versions                    controllers.EditionsController.getVersions(id)
 POST        /editions-api/issues/:id/publish                     controllers.EditionsController.publishIssue(id)
 GET         /editions-api/issues/:id/preview                     controllers.EditionsController.getPreviewEdition(id)
+GET         /editions-api/issues/:id/preflightchecks             controllers.EditionsController.checkIssue(id: String)
 
```

Comment:
`/editions-api/issues/:id/preflightchecks`
to
`/editions-api/issues/:id/preflight-checks`

Reply:
Done